src/cmd/INIT/package.sh

```diff
@@ -31,6 +31,7 @@ esac
 checksh()
 {
 	egrep 'Version.*(88|1993)' $1 >/dev/null 2>&1 ||
+	$1 -c '(( 2 + 2 ))' >/dev/null 2>&1 &&
```

Comment:
This command should be preceded by a `PATH=/dev/null` assignment to avoid executing a possible external command `2`.